app/models/solidus_stripe/gateway.rb

```diff
@@ -48,7 +48,13 @@ def authorize(amount_in_cents, source, options = {})
         success: true,
         message: "PaymentIntent was confirmed successfully",
         response_code: stripe_payment_intent.id,
-        data: stripe_payment_intent,
+        data: stripe_payment_intent
```

Comment:
super nit: leaving the final command will exclude the line from the diff, since it didn't actually change